back-end/legeno-around-here/src/test/java/wooteco/team/ittabi/legenoaroundhere/acceptance/UserAcceptanceTest.java

```diff
@@ -105,9 +105,33 @@ void manageUser() {
         // 비밀번호 수정 실패 - 동일한 비밀번호로 수정
         assertThatThrownBy(() -> changeMyPassword(accessToken, TEST_USER_OTHER_PASSWORD));
 
+        // 토큰을 가진 유저가 정상적으로 로그인 한 유저인지 확인
+        assertThat(validateUserExists(accessToken)).isTrue();
+
         // 회원 탈퇴
         deleteUser(accessToken);
         findNotExistUser(accessToken);
+
+        // 토큰을 가진 유저가 정상적으로 로그인 한 유저가 아님을 확인
+        assertThat(validateUserExists(accessToken)).isFalse();
+    }
+
+    private boolean validateUserExists(String accessToken) {
+        Integer statusCode = given()
+            .header("X-AUTH-TOKEN", accessToken)
+            .when()
+            .get("/auth")
+            .then()
+            .extract()
+            .statusCode();
+        if (statusCode.equals(HttpStatus.NO_CONTENT.value())) {
+            return true;
+        }
+        if (statusCode.equals(HttpStatus.INTERNAL_SERVER_ERROR.value())) {
+            // Todo: 추후 FORBIDDEN 응답하도록 수정하면 좋을것같음
+            return false;
+        }
+        throw new IllegalStateException("예상 응답코드에서 벗어났습니다.");
     }
```

Comment:
1. 위치 자체가 아래로 이동했으면 좋겠습니다.
   인수 테스트의 특성상, 인수 테스트(`@Test`)들이 위쪽에 나열되는 게 좋다고 생각합니다.

2. `isValidUserExists`와 `isInvalidUserExists` 메서드를 각각 나누면 좋을 것 같습니다. 1번과 같은 맥락에서 메서드 분리된 내용을 확인하지 않고 메서드 명만 확인했을 때 바로 읽히는 코드가 좋다고 생각하기 때문입니다.
```suggestion
    private void isValidUserExists(String accessToken) {
        given()
            .header("X-AUTH-TOKEN", accessToken)
            .when()
            .get("/auth")
            .then()
            .statusCode(HttpStatus. NO_CONTENT.value());
    }

    private void isInvalidUserExists(String accessToken) {
        given()
            .header("X-AUTH-TOKEN", accessToken)
            .when()
            .get("/auth")
            .then()
            .statusCode(HttpStatus. INTERNAL_SERVER_ERROR.value());
    }
```

Reply:
저도 2번 제이미 의견에 동의합니다~